packages/site/next.config.js

```diff
@@ -36,7 +38,15 @@ module.exports = {
     // Swaps out Buble for a smaller version that removes the latest Regex spec features.
     // See https://github.com/FormidableLabs/react-live#what-bundle-size-can-i-expect
     config.plugins.push(
-      new webpack.NormalModuleReplacementPlugin(/^buble$/, require.resolve('@philpl/buble'))
+      new webpack.NormalModuleReplacementPlugin(/^buble$/, require.resolve('@philpl/buble')),
+      new CopyPlugin({
+        patterns: [
+          {
+            from: path.posix.resolve(path.posix.dirname(require.resolve('shiki')), '..'),
```

Comment:
potentially another windows issue

I'm assuming we are going to go down the container route for local development.